packages/flutter/lib/src/material/dialog.dart

```diff
@@ -1129,3 +1140,71 @@ double _paddingScaleFactor(double textScaleFactor) {
   // a non-scaled padding of 24 will produce a padding between 24 and 8.
   return lerpDouble(1.0, 1.0 / 3.0, clampedTextScaleFactor - 1.0)!;
 }
+
+// Generate a DialogTheme that represents the M2 default values.
+// This was generated by hand from the previous hand coded defaults
+// for M2. It uses get method overrides instead of properties to
+// avoid computing values that we may not need upfront.
+class _M2Defaults extends DialogTheme {
```

Comment:
I have switched over to using `_DefaultsM2` for this in other classes. Not critical, but it would be nice to be consistent.